packages/openapi-types/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "openapi-types",
-  "version": "7.2.3",
+  "version": "8.0.0",
```

Comment:
please remove changes to this file.  lerna will bump.

Reply:
Done.